homeassistant/components/fan/__init__.py

```diff
@@ -50,28 +50,28 @@
 ATTR_OSCILLATING = "oscillating"
 ATTR_DIRECTION = "direction"
 
-PROP_TO_ATTR = {
+PROP_TO_ATTR: dict = {
     "speed": ATTR_SPEED,
     "speed_list": ATTR_SPEED_LIST,
     "oscillating": ATTR_OSCILLATING,
     "current_direction": ATTR_DIRECTION,
-}  # type: dict
+}
 
-FAN_SET_SPEED_SCHEMA = ENTITY_SERVICE_SCHEMA.extend(
+FAN_SET_SPEED_SCHEMA: dict = ENTITY_SERVICE_SCHEMA.extend(
```

Comment:
hmm these are not dictionaries. The result of `extend` on a voluptuous schema is `vol.Schema`